SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
```

Comment:
I'm partly speaking as a non-Arch user, but the "while you are at it" just seems to add ambiguity. Docs should ideally be really clear and not too wishy-washy. No strong feelings though, it's *ok*.